source/declarations.tex

```diff
@@ -216,8 +216,9 @@
 A \grammarterm{nodeclspec-function-declaration} shall declare a
 constructor, destructor, or conversion function.
 \begin{note}
-A \grammarterm{nodeclspec-function-declaration} can only be used in a
-\grammarterm{template-declaration}\iref{temp.pre},
+Because a member function cannot be subject to a non-defining declaration
+outside of its class definition\iref{class.mfct}, a \grammarterm{nodeclspec-function-declaration}
```

Comment:
Hm, the first part of this is not strictly true: a member function can also be subject to a non-defining friend declaration. But a nodeclspec-function-declaration can't appear at class scope and in any case `friend` is a decl-specifier.  Perhaps:

> Because a non-defining declaration of a member function can only appear at class scope, a nodeclspec-function-declaration can only be used in a ...

Reply:
Or replace "its class definition" with "a class definition"?

Reply:
That's simpler, PTAL?

Reply:
Going with the simpler solution.